packages/components/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -276,3 +276,50 @@ WithDivider.parameters = {
     },
   },
 };
+
+const Foo = () => (
+  <Box.div
+    borderColor="colorBackgroundComplete"
+    borderStyle="borderStyleSolid"
+    borderWidth="borderWidth10"
+    h="100%"
+    w="100%"
+  >
+    <h3>Foo</h3>
+    Lorem ipsum dolor sit, consectetur. Aliquam ac a. Ligula at et, sodales vel
+    purus.
+  </Box.div>
+);
+
+const Bar = () => <div>Bar</div>;
+
+export const Flexed = (): React.JSX.Element => {
+  const [selectedTab, setSelectedTab] = React.useState("foo");
+
+  return (
+    <Box.div display="flex" h="500px">
+      <Tabs flexed>
+        <TabList aria-label="Page tabs" withDivider>
+          <Tab id="foo" onClick={() => setSelectedTab("foo")}>
+            Foo
+          </Tab>
+          <Tab id="bar" onClick={() => setSelectedTab("bar")}>
+            Foo
+          </Tab>
+        </TabList>
+        <Box.div display="flex" flexDirection="column" flexGrow="1">
+          {selectedTab === "foo" ? <Foo /> : <Bar />}
```

Comment:
You don't need to do this if you use the TabPanels component and have an initialTabId in the `<Tabs />` component. To keep it consistent and use the entire power the Tabs component provides us, we need to use the `TabPanels` component. In this case:

```
<TabPanels>
  <TabPanel>
    <Foo />
  </TabPanel>
  <TabPanel>
    <Bar />
  </TabPanel>
/TabPanels>
```
In this way, you don't need to set an onClick to the Tab component.

Reply:
@apvale :point_up:

Reply:
Good point. I hadn't thought about that. Two things:

1. Pluto stories should be generic and cover the scenario you are adding, so, the flexed tab. So, for me, you can use the TabPanels and TabPanel here to exemplify the new prop. 
2. The issue you mentioned above needs to be fixed in a different way by adding a new prop, for example, to render a TabPanel content only when tab is active. 

For the scenario you mentioned, you can continue to use TabPanels and render the TabPanel content in a certain condition. Like:

```
<TabPanels>
  <TabPanel>
    {condition && <Component />}
  </TabPanel>
</TabPanels>
```

Reply:
ChakraUI has a good example of it that we can try to do for Pluto: https://chakra-ui.com/docs/components/tabs#lazily-mounting-tab-panels